src/nvim/api/win_config.c

```diff
@@ -938,8 +938,8 @@ static void parse_border_style(Object style, FloatConfig *fconfig, Error *err)
   }
 }
 
-static bool parse_float_config(Dict(float_config) *config, FloatConfig *fconfig, bool reconf,
-                               bool new_win, Error *err)
+static bool parse_float_config(win_T *wp, Dict(float_config) *config, FloatConfig *fconfig,
+                               bool reconf, bool new_win, Error *err)
```

Comment:
It seems that `new_win` is no longer needed as it can be replaced by `wp == NULL`.